apps/web/app/(use-page-wrapper)/settings/platform/plans/page.tsx

```diff
@@ -12,7 +18,10 @@ export const generateMetadata = async () => {
   );
 };
 
-const ServerPageWrapper = () => {
+const getData = withAppDirSsr(getServerSideProps);
+
+const ServerPageWrapper = async ({ params, searchParams }: PageProps) => {
+  await getData(buildLegacyCtx(await headers(), await cookies(), await params, await searchParams));
```

Comment:
As I mentioned in a comment below, we want to remove usages of `withAppDirSsr` from every page.tsx ultimately. So instead of using this, let's write logic directly in this RSC! TLDR: (1) remove getServerSideProps file (2) write logic here directly